src/Zeega/AdminBundle/Resources/views/edit_custom.html.twig

```diff
@@ -2,8 +2,12 @@
 
 {% block actions %}
     <div class="sonata-actions">
+<<<<<<< HEAD
```

Comment:
This little merge conflict snippet should definitely be removed before a merge! Same goes for the change on line 7 and 10.  Also, why was this title changed from "Live Link" to "Archive Link"?